test/controllers/clinics_controller_test.rb

```diff
@@ -21,6 +21,16 @@ class ClinicsControllerTest < ActionDispatch::IntegrationTest
     end
   end
 
+  describe 'naf_status' do
+    before {
```

Comment:
You can simplify this a bit -- `create` lets you pass in a hash of stuff as an argument, and it'll set the object attributes accordingly. I think something like `before { @clinic = create :clinic, accepts_naf: true }` should fly.